packages/noise/package.json

```diff
@@ -17,39 +17,24 @@
     "dev": "tsup --watch",
     "release": "npm publish --access=public",
     "start": "npx serve .",
-    "test": "forge test"
+    "test": "forge test && tsc --noEmit"
   },
   "dependencies": {
-    "@types/big-integer": "^0.0.31"
+    "abdk-libraries-solidity": "^3.0.0"
   },
   "devDependencies": {
-    "@flatten-js/interval-tree": "^1.0.19",
-    "@graphprotocol/graph-ts": "^0.27.0",
-    "@magic-works/commonjs-import.meta": "^1.0.1",
     "@nomicfoundation/hardhat-chai-matchers": "^1.0.3",
     "@nomiclabs/hardhat-ethers": "^2.1.1",
     "@types/chai": "^4.3.3",
-    "@types/mocha": "^9.1.1",
     "@types/node-fetch": "^2.6.2",
-    "abdk-libraries-solidity": "^3.0.0",
     "assemblyscript": "^0.21.2",
-    "base64-sol": "^1.1.0",
-    "big-integer": "^1.6.51",
     "chai": "^4.3.6",
     "ds-test": "https://github.com/dapphub/ds-test.git#c9ce3f25bde29fc5eb9901842bf02850dfd2d084",
     "ethers": "^5.7.2",
-    "forge-std": "https://github.com/foundry-rs/forge-std.git#b4f121555729b3afb3c5ffccb62ff4b6e2818fd3",
```

Comment:
i'd prefer to keep `forge-std` installed (and use it for `forge-std/Test.sol` in tests instead of `ds-test/test.sol`), because it makes it easier to use `console.log` if needed without having to first install `forge-std` (and figuring out the right commit hash)